src/keywords.cpp

```diff
@@ -152,6 +153,7 @@ static const chunk_tag_t keywords[] =
    { "else",             CT_PP_ELSE,       LANG_ALL | FLAG_PP                                                          }, // PAWN
    { "elseif",           CT_PP_ELSE,       LANG_PAWN | FLAG_PP                                                         }, // PAWN
    { "emit",             CT_PP_EMIT,       LANG_PAWN | FLAG_PP                                                         }, // PAWN
+   { "endasm",           CT_PP_ENDASM,     LANG_ALL | FLAG_PP                                                          },
```

Comment:
I don't see this one used in the general change set . Do you still need it here?